packages/cli-repl/src/cli-repl.ts

```diff
@@ -43,6 +43,7 @@ import { promisify } from 'util';
 import { getOsInfo } from './get-os-info';
 import { UpdateNotificationManager } from './update-notification-manager';
 import { markTime } from './startup-timing';
+import { SampledAnalytics } from '@mongosh/logging/lib/analytics-helpers';
```

Comment:
This is probably not the import you were going for? `SampledAnalytics` should be exported from the `@mongosh/logging` package itself

Reply:
You are right, fixing it.